arrow/test/test_arrow.py

```diff
@@ -100,28 +101,33 @@ def load_state(filename):
     duration = 1
 
     # semi-quantitative rate constants
-    rates = np.full(n_reactions, 1000)
+    rates = np.full(n_reactions, 1000.0)
 
-    return stoichiometric_matrix, rates, initial_state, final_state
+    return (stoichiometric_matrix, rates, initial_state, final_state)
 
 def complexation_test(make_system):
-    stoichiometric_matrix, rates, initial_state, final_state = load_complexation()
+    stoichiometric_matrix, rates, initial_state, final_state = load_complexation(prefix='beginning')
     duration = 1
 
-    system = make_system(stoichiometric_matrix, rates)
+    system = make_system(stoichiometric_matrix, rates, random_seed=np.random.randint(2**31))
     result = system.evolve(duration, initial_state)
 
     time = np.concatenate([[0.0], result['time']])
     events = result['events']
     occurrences = result['occurrences']
     outcome = result['outcome']
 
+    print(outcome)
+    print(np.where(outcome < 0))
```

Comment:
Printing unlabeled values is not great. Should it instead assert something, directly or via `np.assert...`?

Reply:
Ah, missed those, not intended for production.